examples/mayavi/interactive/lorenz_ui.py

```diff
@@ -108,7 +109,7 @@ def get_vel(self, comp):
             g = scipy.__dict__
             x, y, z = self.points
             s, r, b = self.s, self.r, self.b
-            val = eval(func_str, g,
+            val = ast.literal_eval(func_str, g,
```

Comment:
This is incorrect and will not work, in cases like this `eval` is really better than `ast.literal_eval`.

Reply:
Thank you for your comment 👍